types.go

```diff
@@ -47,14 +47,18 @@ type IceServer struct {
 	URLs []string `json:"urls"`
 }
 
+type UserInfo interface {
+	Username() string
+}
+
 type SignalingServer interface {
 	// All endpoints in the SignalingServer return the (possibly modified)
 	// response from the Host Orchestrator and the status code if it was
 	// able to communicate with it, otherwise it returns an error.
-	NewConnection(msg NewConnMsg) (*SServerResponse, error)
-	Forward(id string, msg ForwardMsg) (*SServerResponse, error)
-	Messages(id string, start int, count int) (*SServerResponse, error)
-	ServeDeviceFiles(devId string, path string, w http.ResponseWriter, r *http.Request) error
+	NewConnection(msg NewConnMsg, user UserInfo) (*SServerResponse, error)
+	Forward(id string, msg ForwardMsg, user UserInfo) (*SServerResponse, error)
+	Messages(id string, start int, count int, user UserInfo) (*SServerResponse, error)
```

Comment:
should we encapsulate this parameters into a new MessagesRequest struct to avoid the long list of parameters, the same for Forward and ServeDeviceFiles?

Reply:
Did for SeveDeviceFiles.

Messages and Forward only have 4 and 3 parameters each and the lines are not that long